packages/client/lib/sync/fetcher/fetcher.ts

```diff
@@ -348,8 +368,8 @@ export abstract class Fetcher<JobTask, JobResult, StorageItem> extends Readable
       encoding: string | null,
       cb: Function
     ) => {
+      const jobItems = job instanceof Array ? job : [job]
```

Comment:
Would it makes sense to do `Array.isArray(job)` instead of `instanceof` ?

Reply:
i think they are functionally equivalent, any reason for suggestion Array.isArray over instanceof?

Reply:
No strong reason, I just felt it's easier to read